demos/remote-mcp-authkit/package.json

```
@@ -0,0 +1,28 @@
+{
+	"name": "remote-mcp-authkit",
+	"version": "0.0.1",
+	"private": true,
+	"scripts": {
+		"build": "tsc -p .",
+		"cf-typegen": "wrangler types --env example",
+		"deploy": "wrangler deploy",
+		"dev": "wrangler dev",
+		"format": "biome format --write",
+		"lint": "biome lint",
+		"lint:fix": "biome lint --fix",
+		"start": "wrangler dev"
+	},
+	"devDependencies": {
+		"@biomejs/biome": "^1.8.2",
+		"@cloudflare/workers-types": "^4.20250310.0",
+		"@cloudflare/workers-oauth-provider": "^0.0.2",
+		"@modelcontextprotocol/sdk": "^1.7.0",
```

```suggestion
		"@modelcontextprotocol/sdk": "^1.7.0",
		"agents": "^0.0.53",
```

To make sure that it gets included when people start just with this directory